controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
@kwk is it okay if I name the tests TestReorderAbove(), TestReorderBelow(), TestReorderTop(), TestReorderBottom(), TestReorderMultipleWorkItems()?

Because now I am using test.ReorderWorkitemOK(..) in many test functions. So, I cannot name all my functions TestReorderWorkItemOK().